lib/createWorkers.js

```diff
@@ -48,10 +50,37 @@ function createWorkers(handlers, config, options = {}) {
 
   return {
     listen: co.wrap(_listen),
-    close: co.wrap(_close)
+    close: co.wrap(_close),
+    get emitter() {
+      return emitter;
+    },
+    wait(key) {
```

Comment:
For both of these feature, I thought about it, but I prefer to postpone it until we have real-life usecases.